src/test/java/net/sf/jabref/exporter/ExportFormatTest.java

```diff
@@ -0,0 +1,69 @@
+package net.sf.jabref.exporter;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.common.base.Charsets;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.MetaData;
+import net.sf.jabref.logic.journals.JournalAbbreviationLoader;
+import net.sf.jabref.model.database.BibDatabase;
+import net.sf.jabref.model.entry.BibEntry;
+
+public class ExportFormatTest {
+
+    @Before
+    public void setUp() {
+        Globals.prefs = JabRefPreferences.getInstance();
+        ExportFormats.initAllExports();
+        Globals.journalAbbreviationLoader = new JournalAbbreviationLoader(Globals.prefs);
+
+    }
+
+    @Test
+    public void testExportingEmptyDatabaseLayoutBasedFormat() throws Exception {
+        BibDatabase db = new BibDatabase();
+        Map<String, IExportFormat> exportFormats = ExportFormats.getExportFormats();
+        IExportFormat exportFormat = exportFormats.get("html");
+        try {
+            File tmpFile = File.createTempFile("exporttest", "html");
+            tmpFile.deleteOnExit();
```

Comment:
There is a slightly more preferable way to deal with temporary files in form of a rule, see the wiki.